salt/modules/boto_route53.py

```diff
@@ -385,12 +391,15 @@ def get_record(name, zone, record_type, fetch_all=False, region=None, key=None,
             break  # the while True
 
         except DNSServerError as e:
-            # if rate limit, retry:
-            if retry_on_rate_limit and 'Throttling' == e.code:
-                log.debug('Throttled by AWS API.')
-                time.sleep(2)
+            if retry_on_rate_limit:
```

Comment:
Didn't want to break compatibility with states using this, but retry_on_error and error_retries seem like better argument names, since it will not only retry on rate limit now.